codebuild/ci/ci-build.yml

```diff
@@ -0,0 +1,117 @@
+version: 0.2
+
+env:
+  variables:
+    CA_REGION: us-east-1
+    DOMAIN: crypto-tools-internal
+    REPOSITORY: java-esdk-ci
+    NAMESPACE: com.amazonaws
+    PACKAGE: aws-encryption-sdk-java
+    GPG_KEY_SECRET_ID: Maven-GPG-Keys
+    GPG_CRED_SECRET_ID: Maven-GPG-Keys-Credentials
```

Comment:
This, surprisingly, worked. Which is great since it makes our CI more portable. i.e.: An Engineer can create a CodeBuild (CB) Project with access to two Secrets `x` and `y`, and then use the "Start build with override" action or just update the CB Project's environment settings to replace these secrets with another GPG key-credential pair.